tests/test_web_server.py

```diff
@@ -26,13 +26,15 @@
         raise exc
 
     logger = mock.Mock()
-    server = await aiohttp_raw_server(handler, logger=logger)
+    server = await aiohttp_raw_server(handler, logger=logger, debug=False)
```

Comment:
I guess these tests were assuming `debug=False` by default which is not the case as we had a separate `debug=True` test.